### PR TITLE
2.0.1

### PR DESCRIPTION
**Pull request by dholth**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #67
closes #69

### CODE REVIEW

**Comment by dholth** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,28 @@
...
   url: https://github.com/conda/conda-package-handling/archive/{{ version }}.tar.gz
-  sha256: 81e0771b8b10ab5c20678a5da714e36d97e321acfc3e5b304ec2e650ce84734b
+  sha256: 7b10976cd8aa09307c41e9f19795347b44e83d22077519dc8f3cd7778f68755c
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - cph = conda_package_handling.cli:main
```
Comment (beckermr) [inline recipe/meta.yaml:16]: I think this can be noarch python

**Comment by dholth** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
`conda-package-streaming-feedstock` has a `python>=3.9` constraint. Also pypy tests were failing.

**Comment by dholth** [comment]
conda-package-streaming should be changed to be Python 3.7+. The Windows test failures are conda-package-streaming being case-sensitive when checking path-inside-other-path.

**Review by dholth — COMMENTED** [review]
require conda-package-streaming 0.6.1

**Comment by dholth** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,29 @@
...
-  # HACK: case-sensitive mismatch in latest conda-build
-  missing_dso_whitelist:
-    - '*/VCRUNTIME140.dll'   # [win]
-    - MSVCR90.dll     # [win]
+  noarch: python
```
Comment (dbast) [inline recipe/meta.yaml:17]: As this is such an important central package: can we temporarily remove noarch + rerender to see it successful building (+testing) on all platforms + all python versions? This proves that the package itself  plus its dependencies work properly there.
```suggestion
  #noarch: python
```

Once CI parses the all CI runs, I would be fine with making it noarch again in this PR.

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
Looks like those two tests fail multiple times (the second one is skipped in windows):
```
tests/test_api.py::test_api_transmute_tarball_to_conda_v2 FAILED
tests/test_api.py::test_api_transmute_to_conda_v2_contents FAILED 
```

**Comment by dholth** [comment]
I was able to get the tests running on x86 pypy 3.8. It's running into an "imports something from conda, which hasn't been ported to pypy" problem...

**Comment by isuruf** [comment]
> this package might not need to run on pypy, when conda doesn't? 

conda does. We've had a miniforge-pypy3 distribution for a couple of years now.

**Comment by dbast** [comment]
conda-package-handling has the functionality to extract, create and transmute packages. Afaik conda only uses the extract feature and but conda-build (which also has pypy versions) uses the create/transmute features of the package.

**Comment by dholth** [comment]
Looks like the errors are straightforward. Missing with `open(path):` statements, `pytest-xprocess` had trouble starting, and possibly, differences in when generators are prepared.

**Comment by dholth** [comment]
https://github.com/conda/conda-package-streaming/pull/48

**Comment by dholth** [comment]
conda-package-handling 2.0.1 awaits review, only changes the dependency. https://github.com/conda/conda-package-handling/pull/162

**Comment by dholth** [comment]
@conda-forge--admin, please restart ci

**Comment by isuruf** [comment]
Tests pass. Thanks.

Let's also wait for https://github.com/conda-forge/zstandard-feedstock/issues/43 to be resolved.

**Comment by dholth** [comment]
@isuruf what's up with some of the azure pipelines?

**Comment by isuruf** [comment]
It's only the `osx-arm64` builds that fail. They are cross compiled and without `cross_python-{{ target_platform }}` building fails. It's fine for them to fail now. (We won't be able to run the tests anyway). When this package becomes `noarch: python` we'll only build on linux.

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Review by dbast — APPROVED** [review]

**Comment by dholth** [comment]
Thanks. Since this is a holiday week, I'd like to put off releasing this package until 28 November or a little later.

**Comment by jakirkham** [comment]
Given the holiday has passed (and it is now Nov 29), are we ready to give this a try?

**Comment by dholth** [comment]
Yes

**Pull request merged**